### ISSUE TITLE
Use for pagination only

### ISSUE THREAD

**Issue by mewejo** [protonemedia/laravel-cross-eloquent-search#3]
Hi, and thanks for the great lib!

I was hoping to just use it for pagination only, but it's throwing an exception with no search terms. Removing that exception, it works fine - but I wondered if there is a reason for enforcing search terms, and if you think allowing pagination only would be possible?

I'm currently getting around it by searching the `created_at` column for `%-` which returns all results, and allows pagination only.#

Happy to contribute a PR, but wanted to check there wasn't a reason against it first. I've not actually looked how you're doing multi-model pagination.

**Comment by pascalbaljet** [comment]
Thanks, that's a great use-case! I would introduce an `allowEmptySeachQuery()` method, something like:

```php
Search::add(Post::class, 'title')
    ->add(Video::class, 'title')
    ->allowEmptySeachQuery()
    ->paginate()
    ->get();
```

And then in the `initializeTerms()` method, add an additional check.

```php
if ($this->terms->isEmpty() && !$this->allowEmptySearchQuery)
```

https://github.com/protonemedia/laravel-cross-eloquent-search/blob/f237df34efa7a4f0e74093eba1a495dd72089c94/src/Searcher.php#L183-L185

I think that should work. In the `Searcher` instance, the `$this->terms` collection would be empty, so no where clauses should be added to the query. Feel free to submit a PR :)

**Comment by pascalbaljet** [comment]
I fixed this in v1.4.0 👍 

https://github.com/protonemedia/laravel-cross-eloquent-search#getting-results-without-searching

**Comment by mewejo** [comment]
Ah, sorry! I was going to implement this, I just hadn't had a chance, sorry @pascalbaljet, and thank you! :)
